frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -124,31 +138,29 @@ function getParameterPanelBorderColor(theme?: DisplayTheme) {
 export const ParametersWidgetContainer = styled(FullWidthContainer)<{
   embedFrameTheme?: DisplayTheme;
   hasScroll: boolean;
+  canSticky: boolean;
   isSticky: boolean;
 }>`
   padding-top: ${space(1)};
   padding-bottom: ${space(1)};
 
   ${props =>
-    props.hasScroll &&
```

Comment:
This doesn't really work. We need to apply the border-bottom only when the filters is sticky. If users use iframe resizer the iframe would fit the dashboard size that makes the filters non-sticky because you couldn't scroll inside the iframe.

I change to detect the sticky filters with IntersectionObserver instead and pass that flag as `isSticky` below.